syntax = "proto3";

message GroupChat {
    int32 fromId = 1;
    int32 toId = 2; // ç¾¤id
    string msg = 3;
}